kolibri/core/errorreports/tasks.py

```diff
@@ -0,0 +1,61 @@
+import logging
+
+import requests
+from django.db import connection
+from django.http import JsonResponse
+from requests.exceptions import ConnectionError
+from requests.exceptions import RequestException
+from requests.exceptions import Timeout
+
+from .models import ErrorReports
+from kolibri.core.tasks.decorators import register_task
+from kolibri.core.utils.urls import join_url
+
+logger = logging.getLogger(__name__)
+
+
+def serialize_error_reports_to_json_response(errors):
+    errors_list = []
+    for error in errors:
+        errors_list.append(
+            {
+                "error_from": error.error_from,
+                "error_message": error.error_message,
+                "traceback": error.traceback,
+                "first_occurred": error.first_occurred,
+                "last_occurred": error.last_occurred,
+                "sent": error.sent,
+                "no_of_errors": error.no_of_errors,
+            }
+        )
+    return JsonResponse(errors_list, safe=False)
```

Comment:
Why are you using a JsonResponse here? It seems like what you actually want to do is use the DjangoJSONEncoder here instead: https://docs.djangoproject.com/en/3.2/topics/serialization/#djangojsonencoder

(note: I understand why this is useful, as it has built in support for serializing datetime objects to JSON, which the `json` library by itself does not automatically handle - but we should be explicit about what we are doing and just do serialization here).